### ISSUE TITLE
Cannot parse config file: /g' | grep -v origin | head -1; };f

### ISSUE THREAD

**Issue by saveman71** [PawelLipski/git-machete#1277]
# Overview

```
git machete traverse                          
Cannot parse config line: /g' | grep -v origin | head -1; };f.

Consider posting an issue at https://github.com/VirtusLab/git-machete/issues/new
```

The offending line in config is a git alias:

```
[alias]
	parent = "!f() { git log --graph --simplify-by-decoration --skip=1 --max-count 1 --format='%D' | sed $'s/, /\\\n/g' | grep -v origin | head -1; };f"
```

which git executes and parse without issues.

Commenting the line is enough to make machete work again

## `git` and `git machete` versions

```
git version 2.45.2
git-machete version 3.26.0
```

**Comment by PawelLipski** [comment]
Ouch, this is probably due to `\n` inside the alias. Good catch, I'll get that fixed soon

**Comment by PawelLipski** [comment]
Soon to be released as v3.26.2
